uk_bin_collection/uk_bin_collection/councils/SouthwarkCouncil.py

```
@@ -39,39 +67,35 @@ def parse_data(self, page: str, **kwargs) -> dict:
-
-            dict_data = {
-                "type": recycling_title,
-                "collectionDate": datetime.strptime(
-                    recycling_next_collection, "%a, %d %B %Y"
-                ).strftime("%d/%m/%Y"),
-            }
-            data["bins"].append(dict_data)
+            if recycling_next_collection:
+                dict_data = {
+                    "type": recycling_title,
+                    "collectionDate": datetime.strptime(
```

Ruff (0.14.0)

`datetime` may be undefined, or defined from star imports

(F405)